defaults/main.yml

```diff
@@ -1,6 +1,5 @@
 ---
 # defaults file for ansible-role-lustre_client
-lustre_client_enabled: False
```

Comment:
This might be confusing but we do have users that do not wish to enable this role even though it is called from playbook. The rationale is that we provide general template for multiple cluster sites. Some do not use Lustre (or atleast this role). Setting this up for mixed environment was easiest by defining this enabled=false as default. So, the functionality should stay here. It can be replaced by using other parameters like lustre_mounts to check if this is enabled.

Reply:
wouldn't make more sense to do this outside the internals of the role?
```
vars:
   lustre_client_enable: false
roles:
...
  - role: lustre_client
    when: lustre_client_enable
```

Reply:
Well, I guess it is a matter of taste. But we seem to use that kind of approach already in the master playbooks. So we can adopt the same here. Does make some parts easier in this role.